File: 0217-contains-duplicate/0217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    // let unique = nums.filter((num,index,arr) => 
    //     (arr.indexOf(num) === index )
    // )
    // if(unique.size !== nums.length){
    //     return true
    // }
    // else{
    //     return false
    // }


    let unique =new Set(nums)
    return unique.size !== nums.length
};
console.log(containsDuplicate([1,2,3]))

File: 0217-contains-duplicate/0217-contains-duplicate.cs
public class Solution {
    public bool ContainsDuplicate(int[] nums) {
    Dictionary<int, int> store = new Dictionary<int, int>();
            for(int i = 0 ; i < nums.Length ; i++){
            if(store.ContainsKey(nums[i])){
                return true;
            }else{
            store[nums[i]] = 1;
            }
        }
        return false;
    }
}